### DISCUSSION TITLE
stackoverflow.com

### DISCUSSION THREAD

**Discussion by pocco81** [catppuccin/catppuccin#1856] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Additional context 📷 (optional)**
I need it, you need it, we need it. I think that the easiest way to achieve this is with Stylus.

**Comment by unseen-ninja** [comment]
![](https://c.tenor.com/vEWTKdXfIqsAAAAC/oh-yeah-mochi.gif)

**Comment by GlowingUmbreon** [comment]
![image](https://user-images.githubusercontent.com/37640997/175189127-9f804f74-6fc9-409f-91b5-52b61e6e9a6d.png)

![image](https://user-images.githubusercontent.com/37640997/175189223-32ab061a-250d-4802-be77-ea2c4da52572.png)

**Reply by uncenter** [reply]
@GlowingUmbreon any interest in picking this back up in https://github.com/catppuccin/userstyles?

**Comment by sgoudham** [comment]
Closing as further discussion/review should take place over at https://github.com/catppuccin/userstyles/pull/549
